src/common/entity/compute_state_display.ts

```diff
@@ -63,7 +63,7 @@ export const computeStateDisplay = (
 
   if (domain === "humidifier") {
     if (compareState === "on" && stateObj.attributes.humidity) {
-      return `${stateObj.attributes.humidity}%`;
+      return `${stateObj.attributes.humidity} %`;
```

Comment:
To be consistent with units of measurement that are not hard-coded (and that always have a blank after the value).